table.csv

```diff
@@ -122,6 +122,7 @@ p384-pub,                       key,            0x1201,         P-384 public Key
 p521-pub,                       key,            0x1202,         P-521 public Key
 ed448-pub,                      key,            0x1203,         Ed448 public Key
 x448-pub,                       key,            0x1204,         X448 public Key
+kangarootwelve,                 multihash,      0x1d01,         fast and secure extendable-output hash function from the Keccak family
```

Comment:
OK moving on then with this, a minor nit, aside from capitalisation, my suggestion for the description here:

> KangarooTwelve is an extendable-output hash function based on Keccak-p

Reply:
Understood, I just wanted to mention why it makes sense to include in the first place (the performance), but am in no way insisting upon keeping it.

And what about capitalisation? None of the names in the `name` column are capitalised, and the `description`s don't seem to start with a capital letter either?

Reply:
Just suggesting the change for the `description` field (starting with "KangarootTwelve ..."). Most of the descriptions (where they exist), it's not totally consistent but let's go with the majority; it's particularly apt when naming something that's been capitalised by the author though so that's easy.